source-map-support.js

```diff
@@ -21,6 +21,10 @@ function isInBrowser() {
   return ((typeof window !== 'undefined') && (typeof XMLHttpRequest === 'function'));
 }
 
+function hasGlobalProcess() {
```

Comment:
Can't we find a better name?
Because it tests more than just the presence of `process`.